client/driver/qemu_test.go

```diff
@@ -121,34 +134,36 @@ func TestQemuDriverUser(t *testing.T) {
 		t.Parallel()
 	}
 	ctestutils.QemuCompatible(t)
-	tasks := []*structs.Task{{
-		Name:   "linux",
-		Driver: "qemu",
-		User:   "alice",
-		Config: map[string]interface{}{
-			"image_path":  "linux-0.2.img",
-			"accelerator": "tcg",
-			"port_map": []map[string]int{{
-				"main": 22,
-				"web":  8080,
-			}},
-			"args": []string{"-nodefconfig", "-nodefaults"},
-			"msg":  "unknown user alice",
-		},
-		LogConfig: &structs.LogConfig{
-			MaxFiles:      10,
-			MaxFileSizeMB: 10,
-		},
-		Resources: &structs.Resources{
-			CPU:      500,
-			MemoryMB: 512,
-			Networks: []*structs.NetworkResource{
-				{
-					ReservedPorts: []structs.Port{{Label: "main", Value: 22000}, {Label: "web", Value: 80}},
+	tasks := []*structs.Task{
+		{
+			Name:   "linux",
+			Driver: "qemu",
+			User:   "alice",
+			Config: map[string]interface{}{
+				"image_path":        "linux-0.2.img",
+				"accelerator":       "tcg",
+				"graceful_shutdown": false,
```

Comment:
Adding a graceful_shutdown test would be nice. Maybe copy StartOpen_Wait, but don't worry about the Open part (that's the restore path)?

Reply:
(Done! Thanks!)